package.json

```diff
@@ -7,9 +7,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/modulesio/archae.git"
+    "url": "https://github.com/toomu/archae.git"
```

Comment:
Yeah, but with this change the whole repo would point to `toomu/archae` from now on in npm. I'm really not sure what the point of that would be.

If this is for testing, it probably shouldn't be committed, or at least should not be part of a PR.

Though I'm not quite sure how this helps in testing in the first place...